sprockets/mixins/http/__init__.py

```diff
@@ -144,6 +144,12 @@ def http_fetch(self, url,
                              method, url, attempt + 1,
                              max_http_attempts, error)
                 continue
+            warning_header = response.headers.get('Warning')
```

Comment:
might be cleaner as:
```python
if 'Warning' in response.headers:
                LOGGER.warning('HTTP Warning Header for %s to %s, '
                                'attempt %i of %i (%s): %s',
                                method, url, response.code, attempt + 1,
                                max_http_attempts, response.headers['Warning'])
```
but happy to merge this was as well.

Reply:
I'd like to keep it as-is to avoid duplicating the `Warning` key.

Reply:
What do you mean duplicate?